index.html

```diff
@@ -23,24 +25,37 @@
       </nav>
 
 
-      <section>
+      <section class="sub">
         <h1>Startrly</h1>
         <sub>A catchy tagline for a great solution</sub>
-        <a href="#">Some Call to Action</a>
+        <br>
+        <br>
+        <p></p>
+        <a href="#"></a>
+        <input name="name" type="text" placeholder="    Some Call To Action">
+      </div>
       </section>
+      <br>
+      <br>
+      <br>
+      <hr>
 
 
       <section>
         <blockquote>
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+          <br>
+          <br>
```

Comment:
content can also be added with out editing HTML page in css using something like:
 blockquote::before{
content: "\\""
}

blockquote::after{
content: "\\""
}